src/hevm/src/EVM/SymExec.hs

```diff
@@ -377,7 +377,7 @@ simplify e = if (mapExpr go e == e)
       | otherwise = Lit 0
     go (EVM.Types.GT a b) = EVM.Types.LT b a
     go (EVM.Types.GEq a b) = EVM.Types.LEq b a
-    go (EVM.Types.LEq a b) = EVM.Types.Not (EVM.Types.GT a b)
+    go (EVM.Types.LEq a b) = EVM.Types.IsZero (EVM.Types.GT a b)
```

Comment:
Bitwise not is not the same as logical not. This rule was incorrect.

Reply:
Woooooow. WHAT JUST HAPPENED. Mind blown. So sorry!

Reply:
haha, I've made that same mistake many times before